neural_modelling/src/neuron/c_main.c

```diff
@@ -272,6 +286,9 @@ void resume_callback(void) {
 //!            executed since start of simulation
 //! \param[in] unused: unused parameter kept for API consistency
 void timer_callback(uint timer_count, UNUSED uint unused) {
+    use(unused);
```

Comment:
I think the marker above means you don't need this

Reply:
marker above? this has been needed for a long time for warning killing in the compiler

Reply:
Note the UNUSED in the line above

Reply:
fixed